node/cli/Cargo.toml

```diff
@@ -63,4 +63,4 @@ tempfile = "3.1.0"
 
 [build-dependencies]
 cli = { package = "substrate-cli", path = "../../core/cli" }
-structopt = "0.2.0"
+structopt = "0.3.3"
```

Comment:
I'm not sure, but shouldn't it be enough to have it in the normal dependencies?

Reply:
No, `build.rs` imports stuff from `structopt`. Just tried and it doesn't build.

Reply:
Yeah I know that, I just thought that build deps are merged with normal deps as well.